scripts/syncup-s3.sh

```diff
@@ -64,6 +64,14 @@ sync_files=(
   scRNA-seq-advanced/data/reference/hs_mitochondrial_genes.tsv
 )
 
+output_files=(
+  scRNA-seq-advanced/data/glioblastoma/normalized/glioblastoma_normalized_sce.rds
+  scRNA-seq-advanced/data/rms/integrated/rms_subset_sce.rds
+)
+
+# combine into one list of files
+sync_files+=(${output_files[@]})
```

Comment:
Yeah, this is fine. I had some thought about having a separate loop for debugging purposes, so we can say if a missing file is from setup or output, but that probably isn't worth the effort & code duplication.